### DISCUSSION TITLE
Config Files

### DISCUSSION THREAD

**Discussion by funkybside** [Jyers/Marlin#246] [Q&A] [answered]
Are the configuration files used for the precompiled binary named "E3V2-BLTouch-5x5-HS-v4.2.2.bin"  available within this repo?   I was unable to locate one that appeared to be aligned to those settings.

**Comment by Jyers** [comment, marked as answer]
For that particular skew you would simply have to take the BL Touch 3x3 HS config template and change GRID_MAX_POINTS_X from 3 to 5
